packages/create-app/index.js

```diff
@@ -76,12 +76,17 @@ async function init() {
 
   // determine template
   let template = argv.t || argv.template
-
-  const availableTemplates = TEMPLATES.map((template) => stripColors(template))
-  const isValidTemplate = availableTemplates.includes(template)
-  const message = isValidTemplate
-    ? `Select a template:`
-    : `${template} isn't a valid template. Please choose from below: `
+  let message = 'Select a template:'
+  let isValidTemplate
```

Comment:
Just to be a bit more explicit
```suggestion
  let isValidTemplate = false
```